inspirehep/config.py

```diff
@@ -1437,7 +1438,7 @@ def _(x):
             },
             "source": {
                 "terms": {
-                    "field": "metadata.acquisition_source.method",
+                    "field": "metadata.acquisition_source.source",
```

Comment:
but isn't `forms` a synonym for `submitter` or `user` or whatever it is we are using?